services/wallet/service.go

```diff
@@ -31,6 +31,7 @@ func NewService(
 		dataSourceType: DataSourceStatic,
 	})
 	tokenManager := &TokenManager{db: db, RPCClient: rpcClient, networkManager: rpcClient.NetworkManager}
+	tokenManager.OverrideTokens()
```

Comment:
I wonder if `OverrideTokens()` should be an internally called function of a new function called `NewTokenManager`. It seems that this function should always be called by default on a new `TokenManager`

Reply:
You are  correct, doing NewTokenManager makes more sense, I will change